src/main/java/com/example/petbuddybackend/service/mapper/CaretakerMapper.java

```diff
@@ -22,12 +26,12 @@ public interface CaretakerMapper {
     CaretakerComplexInfoDTO mapToCaretakerComplexInfoDTO(Caretaker caretaker);
 
     @Mapping(target = "accountData", source = "accountData")
-    Caretaker mapToCaretaker(ModifyCaretakerDTO caretakerDTO, AppUser accountData);
+    Caretaker mapToCaretaker(CreateCaretakerDTO caretakerDTO, AppUser accountData, List<PhotoLink> offerPhotos);
```

Comment:
Pod spodem mapper tworzy nową liste w opiekunie na zdjecia. Jest mała szansa ale moze byc z tym problem potem przez orpahnRemoval, hibernate sie moze pogubic. Kazalbym mapperowi dodac do isteniejacej juz listy

Reply:
Chociaz to jest w builderze wiec moze bedzie git

Reply:
Hm, chociaż od buildera to spodziewałbym się, że jak wrzucam listę to dostaje on tą konkretną listę a nie kopię, więc chyba to poprawię. 

Na pewno będzie to szybsze